Yoast/wordpress-seo
# Close Help Center when clicking Go Premium.

## Summary

This PR can be summarized in the following changelog entry:

* Close the Help Center when clicking the Go Premium link

## Relevant technical choices:

I've initially used a separate function but eslint `no-use-before-define` bothered me. Then changed to "procedural jQuery", and 2 nested `if`. Ugly but it works (at least it's simple). Let me know if you prefer some more elegant way to do this. 

## Test instructions

This PR can be tested by following these steps:

- go in an edit post or edit taxonomy page
- open the Help Center
- click on the Go Premium link at the top right of the metabox
- check the Help Center closes and the "Yoast SEO Premium" tab is displayed correctly 

Fixes #8229
